corehq/apps/ota/tests/test_utils.py

```diff
@@ -24,6 +24,7 @@ class RestorePermissionsTest(LocationHierarchyTestCase):
 
     @classmethod
     def setUpClass(cls):
+        delete_all_users()
```

Comment:
Assuming this was just for when you were testing locally? Looks like this is already in teardown